server/.env.test

```diff
@@ -0,0 +1,5 @@
+JWT_SECRET=secret-jwt
+MEMORY_DB=true
+REGISTRATION_PASSWORD=password
+SECRET_KEY=secret
+DRIFT_HOME=~/.drift-tests
```

Comment:
I wonder if we want to define this in code to the test directory (or a new directory for holding the test db). I can imagine wanting to inspect it and ~/.drift-tests not being intuitive. Thoughts?

Reply:
To be honest, it is not even needed, as `MEMORY_DB` is set to true 😅. I just forgot to remove it. It is removed now